doc/Makefile.am

```diff
@@ -148,6 +148,75 @@ CLEAN_LOCAL_TARGETS += clean-gtkdoc-header-local
 clean-gtkdoc-header-local:
 	-rm -rf xml/ Doxyfile-gi Doxyfile-gi.stamp geany-gtkdoc.h geany-sciwrappers-gtkdoc.h
 
+if HAVE_INTROSPECTION
+
+INTROSPECTION_GIRS  = Geany-1.0-pre.gir GeanyScintilla-1.0-pre.gir
```

Comment:
Why do we need both? The point of the former is to provide a method on the Scintilla object (well as much as C can), if the latter does that, then why have both?